src/xsl/openrosa2html5form.xsl

```diff
@@ -429,16 +478,12 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
 
         <!-- note that bindings are not required -->
         <!--<xsl:variable name="binding" select="/h:html/h:head/xf:model/xf:bind[@nodeset=$nodeset_used] | /h:html/h:head/xf:model/xf:bind[@nodeset=$nodeset]" />-->
-        <xsl:variable name="binding" select="/h:html/h:head/xf:model/xf:bind[@nodeset=$nodeset]" />
+        <xsl:variable name="binding" select="key('nodeset-bindings', $nodeset)" />
 
         <!-- If this is a bind element that also has an input, do nothing as it will be dealt with by the corresponding xf:input -->
         <!-- Note that this test is not fully spec-compliant. It will work with XLS-form produced forms that have no relative nodes
```

Comment:
I've read this comment about a hundred times and can't figure out what "forms that have no relative nodes" means. This does look spec-compliant to me.